app/views/layouts/application.slim

```diff
@@ -15,14 +15,13 @@ html
 
     .container
       - flash.each do |key, value|
-        div class="#{flash_class(key)} alert-dismissable"
+        div class="flash #{flash_class(key)}"
+          span.flash-close.js-flash-close.octicon.octicon-x
           = value
 
       = yield
-
-      = render 'subscribers/form'
-
-      sub
-        br
-          | Powered by&nbsp;
-          a href="https://github.com/rilian/newsletter" rilian/newsletter
+      - unless manager_signed_in?
```

Comment:
I suspect managers will be already subscribers & IMO this form is eyewinker every time I navigate the site.